tests/test_ioc_system.py

```diff
@@ -178,216 +107,94 @@ async def test_create_softioc_update(
 #         purge_channel_caches()
 
 
-@pytest.mark.asyncio
-async def test_create_softioc_record_update_send_to_panda(
-    dummy_server_system: DummyServer,
-    subprocess_ioc,
-):
-    """Test that updating a record causes the new value to be sent to PandA"""
-    # Set the special response for the server
-    dummy_server_system.expected_message_responses.update(
-        {"PCAP1.TRIG_EDGE=Either": "OK"}
-    )
-
-    await caput(TEST_PREFIX + ":PCAP1:TRIG_EDGE", "Either", wait=True, timeout=TIMEOUT)
-
-    # Confirm the server received the expected string
-    assert (
-        "PCAP1.TRIG_EDGE=Either" not in dummy_server_system.expected_message_responses
-    )
-
-
-@pytest.mark.asyncio
-async def test_create_softioc_arm_disarm(
-    dummy_server_system: DummyServer,
-    subprocess_ioc,
-):
-    """Test that the Arm and Disarm commands are correctly sent to PandA"""
-
-    # Set the special response for the server
-    dummy_server_system.expected_message_responses.update(
-        {"*PCAP.ARM=": "OK", "*PCAP.DISARM=": "OK"}
-    )
-
-    await caput(TEST_PREFIX + ":PCAP:ARM", 1, wait=True, timeout=TIMEOUT)
-
```

Comment:
Do we have an equivalent of thses tests?

Reply:
To me it seems like the tests are checking the `dummy_server_system` more than the ioc - asserting the dummy server responses are consumed when the message comes in from the ioc. Since we don't use the dummy server I didn't feel the need to adapt this?

I suppose I could just do the caget, caput?

Reply:
ok, probably don't need these tests then

Reply:
I believe the tests are still very important - they are confirming that updating PVs will send the correct messages to PandA to update the relevant fields/attributes there. 

Note I haven't gone through the rest of this PR yet so may be missing context.

Reply:
@AlexanderWells-diamond

Okay, given the new structure, I have now made a new multiprocessing queue which captures all commands passed to the panda. This will be used to convert these two unit tests